### PR TITLE
Initialisation of tailwindscss + global style + ShowMore button refactoring.

### PR DESCRIPTION
**Pull request by Fuzznimp**
## Description

This PR was done as a peer programming task with @Oluane. After our last discussion on the subject, we decided to give `tailwindcss` a solid test. 

Appart from the initialisation, our goal was to replace our current global style component with the built-in (Preflight)[https://tailwindcss.com/docs/preflight] feature, and check what needed to be fine tuned.

We then checked every pages to see if there was something off. The only component that needed attention was the `ShowMore` button. We decided it was time to refactor it, as there was useless divisions and css rules.

The result is pretty impressive, to say the least 😅. Almost no external documentation was needed, thanks to the `postcss` and `tailwindcss` VSCode extensions. Overall, a very good experience for both of us.

I also fixed the test related to the confirmation box because the forgotten `awaits` were breaking it this branch.

## Type of change

- [x] **Chore** (non-breaking change which refactors / improves the existing code base).
- [ ] **Bug fix** (non-breaking change which fixes an issue).
- [x] **New feature** (non-breaking change which adds functionality).
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected).

## List of added dependencies

- `autoprefixer`: "10.3.1"
- `postcss`: "8.3.6"
- `tailwindcss`: "2.2.7"

## Checklist

- [x] My code follows the style [**contributing guidelines**][contributing_file] of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.

[contributing_file]: https://github.com/fewlinesco/connect-account/blob/master/README.adoc

### CODE REVIEW

File: src/design-system/globals/global-style.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
+@tailwind base;
```
Comment (Fuzznimp) [inline src/design-system/globals/global-style.css:1]: Classic GitHub's spacing being off.

Comment (Fuzznimp) [inline src/design-system/globals/global-style.css:1]: https://unpkg.com/tailwindcss@2.2.7/dist/base.css

File: src/design-system/globals/global-style.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+	h3:not(:first-of-type) {
+    margin: 4rem 0 2rem 0;
+	}
+
+	p {
+		font-size: "1.6rem";
+	}
+}
+
+@tailwind components;
```
Comment (Fuzznimp) [inline src/design-system/globals/global-style.css:46]: https://unpkg.com/tailwindcss@2.2.7/dist/components.css

**Comment by frieda-ewlines** [comment]
<details><summary>Click here to see the Lighthouse manifest</summary><br><p><strong>/:</strong></p><ul><li>Performance: 0.9</li><li>Accessibility: 1</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/:</strong></p><ul><li>Performance: 0.93</li><li>Accessibility: 0.93</li><li>Best practice: 0.93</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/user-profile/new/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 0.93</li><li>Best practice: 0.93</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/:</strong></p><ul><li>Performance: 0.93</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/email/new/:</strong></p><ul><li>Performance: 0.85</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/phone/new/:</strong></p><ul><li>Performance: 0.87</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/:</strong></p><ul><li>Performance: 0.9</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/sudo/:</strong></p><ul><li>Performance: 0.9</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/locale/:</strong></p><ul><li>Performance: 0.9</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr></details>

**Comment by frieda-ewlines** [comment]
<details><summary>Click here to see the Lighthouse manifest</summary><br><p><strong>/:</strong></p><ul><li>Performance: 0.86</li><li>Accessibility: 1</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/:</strong></p><ul><li>Performance: 0.94</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/:</strong></p><ul><li>Performance: 0.92</li><li>Accessibility: 0.93</li><li>Best practice: 0.93</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/user-profile/new/:</strong></p><ul><li>Performance: 0.89</li><li>Accessibility: 0.93</li><li>Best practice: 0.93</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/:</strong></p><ul><li>Performance: 0.89</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/email/new/:</strong></p><ul><li>Performance: 0.88</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/phone/new/:</strong></p><ul><li>Performance: 0.89</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/:</strong></p><ul><li>Performance: 0.93</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/sudo/:</strong></p><ul><li>Performance: 0.89</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/locale/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr></details>

File: src/components/icons/triangle/triangle.tsx [hunk 3]
```diff
@@ -1,37 +1,18 @@
...
-        />
-      </svg>
-    </Rotate>
+    <svg
+      width="11"
+      height="5"
+      viewBox="0 0 11 5"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      className={rotate ? "mt-1 transform rotate-180" : ""}
```
Comment (Fenntasy) [inline src/components/icons/triangle/triangle.tsx:11]: wasn't it a margin-bottom before?

File: src/design-system/globals/global-style.css (hunk 2 above)
Comment (Fenntasy) [reply src/design-system/globals/global-style.css:46]: does this method still purge the unused classes?

File: src/components/icons/triangle/triangle.tsx (hunk 3 above)
Comment (Fuzznimp) [reply src/components/icons/triangle/triangle.tsx:11]: It was, but the value was `0.3rem`, a value not accessible with tailwind. So we changed it around so it was still align centered.

**Comment by frieda-ewlines** [comment]
Task linked: [CU-1cdmdzb [config] - Setup tailwindcss + theme + global style](https://app.clickup.com/t/1cdmdzb)

**Comment by frieda-ewlines** [comment]
<details><summary>Click here to see the Lighthouse manifest</summary><br><p><strong>/:</strong></p><ul><li>Performance: 0.9</li><li>Accessibility: 1</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/:</strong></p><ul><li>Performance: 0.83</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/:</strong></p><ul><li>Performance: 0.84</li><li>Accessibility: 0.93</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/user-profile/new/:</strong></p><ul><li>Performance: 0.83</li><li>Accessibility: 0.93</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/:</strong></p><ul><li>Performance: 0.87</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/email/new/:</strong></p><ul><li>Performance: 0.83</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/phone/new/:</strong></p><ul><li>Performance: 0.84</li><li>Accessibility: 0.92</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/:</strong></p><ul><li>Performance: 0.87</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/sudo/:</strong></p><ul><li>Performance: 0.87</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/locale/:</strong></p><ul><li>Performance: 0.84</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr></details>

**Comment by frieda-ewlines** [comment]
<details><summary>Click here to see the Lighthouse manifest</summary><br><p><strong>/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 1</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/:</strong></p><ul><li>Performance: 0.96</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/:</strong></p><ul><li>Performance: 0.93</li><li>Accessibility: 0.93</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/user-profile/new/:</strong></p><ul><li>Performance: 0.89</li><li>Accessibility: 0.93</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/:</strong></p><ul><li>Performance: 0.93</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/email/new/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/phone/new/:</strong></p><ul><li>Performance: 0.92</li><li>Accessibility: 0.92</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/:</strong></p><ul><li>Performance: 0.88</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/sudo/:</strong></p><ul><li>Performance: 0.92</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/locale/:</strong></p><ul><li>Performance: 0.93</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr></details>

**Comment by frieda-ewlines** [comment]
<details><summary>Click here to see the Lighthouse manifest</summary><br><p><strong>/:</strong></p><ul><li>Performance: 0.88</li><li>Accessibility: 1</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/:</strong></p><ul><li>Performance: 0.87</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/:</strong></p><ul><li>Performance: 0.93</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/user-profile/new/:</strong></p><ul><li>Performance: 0.89</li><li>Accessibility: 0.91</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/:</strong></p><ul><li>Performance: 0.93</li><li>Accessibility: 0.91</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/email/new/:</strong></p><ul><li>Performance: 0.9</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/phone/new/:</strong></p><ul><li>Performance: 0.87</li><li>Accessibility: 0.92</li><li>Best practice: 0.87</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/sudo/:</strong></p><ul><li>Performance: 0.85</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/locale/:</strong></p><ul><li>Performance: 0.9</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr></details>

**Review by Oluane — APPROVED** [review]

**Pull request merged**